src/components/LikeButton.tsx

```diff
@@ -0,0 +1,19 @@
+import { Pressable, Text } from "react-native";
+import { useLikeContext } from "../contexts/LikedContext";
+
+const LikeButton = ({ pokemonId }: { pokemonId?: number }) => {
```

Comment:
Question for you: what is a difference between these two lines. And why your line is incorrect?
```suggestion
const LikeButton = ({ pokemonId }: { pokemonId: number | undefined }) => {
```

Reply:
`pokemonId: number | undefined` required having `pokemonId` which might be undefined my typing allows to use 
`<LikeButton />` didn't know about it :o